### PR TITLE
4038 crash when scrolling contributions

### LINKED ISSUE #4038: Crash when scrolling down my contributions
Latest master 2ed5a74
```
11-16 07:58:29.607 15666 15666 D ViewRootImpl@de9072a[MainActivity]: ViewPostIme pointer 1
11-16 07:58:45.923  5635  5804 I clp-JNI : Add pinfo.  (channel) b15b7b1 fr.free.nrw.commons/fr.free.nrw.commons.contributions.MainActivity (server) (pkg) AppWindowToken{21bd8c5 token=Token{dce033c ActivityRecord{a13012f u0 fr.free.nrw.commons/.contributions.MainActivity t16652}}} 15666 10383 (action) 0
11-16 07:58:45.927 15666 15666 D ViewRootImpl@de9072a[MainActivity]: ViewPostIme key 0
11-16 07:58:45.968  5635  5804 I clp-JNI : Add pinfo.  (channel) b15b7b1 fr.free.nrw.commons/fr.free.nrw.commons.contributions.MainActivity (server) (pkg) AppWindowToken{21bd8c5 token=Token{dce033c ActivityRecord{a13012f u0 fr.free.nrw.commons/.contributions.MainActivity t16652}}} 15666 10383 (action) 1
11-16 07:58:45.969 15666 15666 D ViewRootImpl@de9072a[MainActivity]: ViewPostIme key 1
11-16 07:58:45.969 15666 22165 V OkHttp  : --> GET https://raw.githubusercontent.com/commons-app/campaigns/master/campaigns.json
11-16 07:58:45.971 15666 22165 V OkHttp  : <-- 200 OK https://raw.githubusercontent.com/commons-app/campaigns/master/campaigns.json (1ms, unknown-length body)
11-16 07:58:45.973 15666 15666 D ScrollView:  onsize change changed 
11-16 07:58:45.973 15666 15666 D ScrollView:  onsize change changed 
11-16 07:58:45.989 15666 22173 D OkHttp  : --> GET https://commons.wikimedia.org/w/api.php?format=json&formatversion=2&errorformat=plaintext&action=query&meta=notifications&notformat=model&notlimit=max&notwikis=wikidatawiki%7Ccommonswiki%7Cenwiki&notfilter=%21read
11-16 07:58:45.990 15666 22173 D CookieManager: Domain:commons.wikimedia.org
11-16 07:58:46.018 15666 22082 D OkHttp  : --> GET https://commons.wikimedia.org/w/api.php?action=query&format=json&formatversion=2&generator=allimages&gaisort=timestamp&gaidir=older&prop=imageinfo&iiprop=url|extmetadata&iiurlwidth=640&iiextmetadatafilter=DateTime|Categories|GPSLatitude|GPSLongitude|ImageDescription|DateTimeOriginal|Artist|LicenseShortName|LicenseUrl&gaiuser=Syced&gailimit=10&gaicontinue=20201024151506|Nandi_bear_1961_(cropped).png&continue=gaicontinue||
11-16 07:58:46.018 15666 22082 D CookieManager: Domain:commons.wikimedia.org
11-16 07:58:46.525 15666 22173 D OkHttp  : <-- 200 https://commons.wikimedia.org/w/api.php?format=json&formatversion=2&errorformat=plaintext&action=query&meta=notifications&notformat=model&notlimit=max&notwikis=wikidatawiki%7Ccommonswiki%7Cenwiki&notfilter=%21read (534ms, unknown-length body)
11-16 07:58:46.526 15666 15666 D ContributionsFragment: Number of notifications is 0
11-16 07:58:47.439  5635  5804 I clp-JNI : Add pinfo.  (channel) b15b7b1 fr.free.nrw.commons/fr.free.nrw.commons.contributions.MainActivity (server) (pkg) AppWindowToken{21bd8c5 token=Token{dce033c ActivityRecord{a13012f u0 fr.free.nrw.commons/.contributions.MainActivity t16652}}} 15666 10383 (action) 0
11-16 07:58:47.440 15666 15666 D ViewRootImpl@de9072a[MainActivity]: ViewPostIme pointer 0
11-16 07:58:47.486 15666 15666 W StaticLayout: maxLineHeight should not be -1.  maxLines:2 lineCount:2
11-16 07:58:47.486 15666 15666 W StaticLayout: maxLineHeight should not be -1.  maxLines:2 lineCount:2
11-16 07:58:47.512  5635  5804 I clp-JNI : Add pinfo.  (channel) b15b7b1 fr.free.nrw.commons/fr.free.nrw.commons.contributions.MainActivity (server) (pkg) AppWindowToken{21bd8c5 token=Token{dce033c ActivityRecord{a13012f u0 fr.free.nrw.commons/.contributions.MainActivity t16652}}} 15666 10383 (action) 1
11-16 07:58:47.513 15666 15666 D ViewRootImpl@de9072a[MainActivity]: ViewPostIme pointer 1
11-16 07:58:47.531 15666 17503 I         : Increase max job count 60
11-16 07:58:48.582 15666 22082 D OkHttp  : <-- 200 https://commons.wikimedia.org/w/api.php?action=query&format=json&formatversion=2&generator=allimages&gaisort=timestamp&gaidir=older&prop=imageinfo&iiprop=url|extmetadata&iiurlwidth=640&iiextmetadatafilter=DateTime|Categories|GPSLatitude|GPSLongitude|ImageDescription|DateTimeOriginal|Artist|LicenseShortName|LicenseUrl&gaiuser=Syced&gailimit=10&gaicontinue=20201024151506|Nandi_bear_1961_(cropped).png&continue=gaicontinue|| (2564ms, unknown-length body)
11-16 07:58:48.584 15666 22082 D OkHttp  : --> GET https://commons.wikimedia.org/w/api.php?format=json&action=wbgetentities&props=labels&languagefallback=1&ids=M95178133%7CM95178483%7CM95186315%7CM95187080%7CM95190808%7CM95206194%7CM95223029%7CM95272930%7CM95288857%7CM95410471
11-16 07:58:48.585 15666 22082 D CookieManager: Domain:commons.wikimedia.org
11-16 07:58:48.681  5635  5804 I clp-JNI : Add pinfo.  (channel) b15b7b1 fr.free.nrw.commons/fr.free.nrw.commons.contributions.MainActivity (server) (pkg) AppWindowToken{21bd8c5 token=Token{dce033c ActivityRecord{a13012f u0 fr.free.nrw.commons/.contributions.MainActivity t16652}}} 15666 10383 (action) 0
11-16 07:58:48.682 15666 15666 D ViewRootImpl@de9072a[MainActivity]: ViewPostIme pointer 0
11-16 07:58:48.748  5635  5804 I clp-JNI : Add pinfo.  (channel) b15b7b1 fr.free.nrw.commons/fr.free.nrw.commons.contributions.MainActivity (server) (pkg) AppWindowToken{21bd8c5 token=Token{dce033c ActivityRecord{a13012f u0 fr.free.nrw.commons/.contributions.MainActivity t16652}}} 15666 10383 (action) 1
11-16 07:58:48.749 15666 15666 D ViewRootImpl@de9072a[MainActivity]: ViewPostIme pointer 1
11-16 07:58:49.105 15666 22169 D OkHttp  : --> GET https://en.wikipedia.org/api/rest_v1/page/media-list/Takahan_Ryokan
11-16 07:58:49.106 15666 22169 D CookieManager: Domain:en.wikipedia.org
11-16 07:58:49.106 15666 22169 D CookieManager: Adding centralauth cookies
11-16 07:58:49.227 15666 22169 D OkHttp  : <-- 200 https://en.wikipedia.org/api/rest_v1/page/media-list/Takahan_Ryokan (122ms, 80-byte body)
11-16 07:58:49.295 15666 22082 D OkHttp  : <-- 200 https://commons.wikimedia.org/w/api.php?format=json&action=wbgetentities&props=labels&languagefallback=1&ids=M95178133%7CM95178483%7CM95186315%7CM95187080%7CM95190808%7CM95206194%7CM95223029%7CM95272930%7CM95288857%7CM95410471 (710ms, unknown-length body)
11-16 07:58:49.363 15666 15666 D AndroidRuntime: Shutting down VM
11-16 07:58:49.365 15666 15666 E AndroidRuntime: FATAL EXCEPTION: main
11-16 07:58:49.365 15666 15666 E AndroidRuntime: Process: fr.free.nrw.commons, PID: 15666
11-16 07:58:49.365 15666 15666 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'fr.free.nrw.commons.Media fr.free.nrw.commons.contributions.Contribution.getMedia()' on a null object reference
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at fr.free.nrw.commons.contributions.ContributionViewHolder.init(ContributionViewHolder.java:70)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at fr.free.nrw.commons.contributions.ContributionsListAdapter.onBindViewHolder(ContributionsListAdapter.java:49)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at fr.free.nrw.commons.contributions.ContributionsListAdapter.onBindViewHolder(ContributionsListAdapter.java:14)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:7093)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:7172)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:6037)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6304)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6143)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:6139)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2314)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.GridLayoutManager.layoutChunk(GridLayoutManager.java:560)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1591)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:668)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.GridLayoutManager.onLayoutChildren(GridLayoutManager.java:170)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:4141)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:3853)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4410)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1083)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1812)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1656)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.onLayout(LinearLayout.java:1565)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1083)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1812)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1656)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.onLayout(LinearLayout.java:1565)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1812)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1656)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.onLayout(LinearLayout.java:1565)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1812)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1656)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.LinearLayout.onLayout(LinearLayout.java:1565)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at com.android.internal.policy.DecorView.onLayout(DecorView.java:1041)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.View.layout(View.java:22420)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewGroup.layout(ViewGroup.java:6579)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:3343)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2807)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1853)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:8476)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:949)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.Choreographer.doCallbacks(Choreographer.java:761)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.Choreographer.doFrame(Choreographer.java:696)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:935)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:873)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:214)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7032)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
11-16 07:58:49.365 15666 15666 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
```
Not sure if that matters, but some context: I used Nearby to upload a picture of both items Takahan Ryokan and Nakatani Foundation.

### PR DESCRIPTION
**Pull request by gouri-panda**
Fixes #4038 

What changes did you make and why?
Checks the Media value before accessing it.

**Screenshots (for UI changes only)**
NA

### CODE REVIEW

**Comment by gouri-panda** [comment]
@nicolas-raoul @misaochan  Ping for review.

File: app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java [hunk 1]
```diff
@@ -68,8 +68,9 @@ public void init(final int position, final Contribution contribution) {
     this.contribution = contribution;
     this.position = position;
     titleView.setText(contribution.getMedia().getMostRelevantCaption());
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java:70]: If `contribution.getMedia()` is null then this line fails first, I guess?
More importantly, what are the situations where a null `contribution.getMedia()` makes sense? If there is no situation where this makes sense, better search for the root cause.
Thanks! :-)

Comment (gouri-panda) [reply app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java:70]: @nicolas-raoul I don't know the root cause. Unfortunately, I don't familiar with the API of wikidata. Should I hot fix this for now like  this
```java
String imageSource = "";
    if (contribution.getMedia() != null) {
      titleView.setText(contribution.getMedia().getMostRelevantCaption());
      authorView.setText(contribution.getMedia().getCreator());
      imageSource = chooseImageSource(contribution.getMedia().getThumbUrl(),
        contribution.getLocalUri());
    }
```

Or Should I  familiar with the API?
P.S I can't find a communication channel. Is there a communication channel ? Thanks!

**Comment by ashishkumar468** [comment]
@gouri-panda  Thanks for the PR, we have a beta release impending, are you planning to update the PR anytime soon?

**Comment by gouri-panda** [comment]
@ashishkumar468 sure thanks.

**Comment by nicolas-raoul** [comment]
I just tried this branch.
Wizard runs fine, I can log in fine, but as soon as the gallery tries to load I get this crash:
```
12-18 14:05:34.933  7629  7629 W ree.nrw.common: Accessing hidden method Landroid/graphics/Typeface;->createFromFamiliesWithDefault([Landroid/graphics/FontFamily;Ljava/lang/String;II)Landroid/graphics/Typeface; (light greylist, reflection)
12-18 14:05:34.948  7629  7635 I ree.nrw.common: Compiler allocated 4MB to compile void android.widget.TextView.<init>(android.content.Context, android.util.AttributeSet, int, int)
12-18 14:05:34.968  7629  7886 V OkHttp  : --> GET https://tools.wmflabs.org/urbanecmbot/commonsmisc/uploadsbyuser.py?user=Syced
12-18 14:05:34.968  7629  7886 V OkHttp  : --> END GET
12-18 14:05:34.977  5120  5200 D DnsProxyListener: DNSDBG::dns addrinfo af 2
12-18 14:05:34.982  7629  7886 D TcpOptimizer: TcpOptimizer-ON
12-18 14:05:35.011  7629  7629 D AndroidRuntime: Shutting down VM
12-18 14:05:35.012  7629  7629 E AndroidRuntime: FATAL EXCEPTION: main
12-18 14:05:35.012  7629  7629 E AndroidRuntime: Process: fr.free.nrw.commons, PID: 7629
12-18 14:05:35.012  7629  7629 E AndroidRuntime: java.lang.IllegalStateException: Cannot change whether this adapter has stable IDs while the adapter has registered observers.
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.recyclerview.widget.RecyclerView$Adapter.setHasStableIds(RecyclerView.java:7209)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at fr.free.nrw.commons.contributions.ContributionsListFragment.initRecyclerView(ContributionsListFragment.java:148)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at fr.free.nrw.commons.contributions.ContributionsListFragment.onViewCreated(ContributionsListFragment.java:129)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:322)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1185)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1354)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1432)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1495)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2617)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.dispatchActivityCreated(FragmentManager.java:2569)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.Fragment.performActivityCreated(Fragment.java:2722)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentStateManager.activityCreated(FragmentStateManager.java:336)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1186)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1354)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1432)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1495)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:447)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.executeOps(FragmentManager.java:2167)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1990)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1945)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1847)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2621)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.dispatchActivityCreated(FragmentManager.java:2569)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentController.dispatchActivityCreated(FragmentController.java:247)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:541)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:201)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1391)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.app.Activity.performStart(Activity.java:7348)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3131)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:180)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:165)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:142)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1947)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:214)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7032)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
12-18 14:05:35.012  7629  7629 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
12-18 14:05:35.013  7629  7629 E ACRA    : ACRA caught a IllegalStateException for fr.free.nrw.commons
```

**Comment by gouri-panda** [comment]
@nicolas-raoul we can't  use adapter.setHasStableIds(true) with ObservableArrayList/ObservableFields.
Can we do this  in `ContributionListFragment` ?
```java
if (!adapter.hasObservers()) {
    adapter.setHasStableIds(true)
}
```
@nicolas-raoul @ashishkumar468 Do you have any alternative idea? Thanks!

**Comment by neslihanturan** [comment]
Closing this because a similar solution already is merged  at #4128

**Pull request closed without merging**